features/show_menu.feature

```diff
@@ -0,0 +1,21 @@
+Feature: let_custemers_see_menu
+as a owner.
+to be able to let costumers buy food.
+You want to display a Menu on a page.
+
+
+Background:
+And the following products exist:
+  | Name       | Catagory    | Menu |
```

Comment:
Please don't use capital letters in table headers. That will result in them being used as hash keys and that does not feel like a good practice.